packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -231,7 +235,7 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
 	) {
 		// Back-compat ADO:3455
 		const backCompatRefSeqNum = referenceSequenceNumber ?? this.lastProcessedSequenceNumber;
-		const messagePartial: Omit<IDocumentMessage, "clientSequenceNumber"> = {
+		const messagePartial: Omit<IMessageWithBatchMetadata, "clientSequenceNumber"> = {
```

Comment:
I don't know enough about this area to know the right answer, but this plus the IMessageWithBatchMetadata definition above feels like we're missing a type to describe this situation accurately.

Reply:
we are. wait until you see containerRuntime. but i think in this case the local type is fine, and give a bit more safetly than before.